### PR TITLE
Added version to setup.cfg.

### PR DESCRIPTION
**Pull request by cmccully**
I have added the version number to setup.cfg in hopes of making it more intuitive to find.

### CODE REVIEW

**Comment by embray** [comment]
This might cause a problem for the astropy release scripts, which I think at least a couple people have used for affiliated packages too.  That said, the release scripts could be fixed to support this which would be fine.

**Comment by astrofrog** [comment]
If we do this, then there are no longer any package-specific variables needed in ``setup.py`` which would be nice, so I'd be in favor.

**Comment by cmccully** [comment]
Any updates on this?

**Comment by mwcraig** [comment]
Thanks for the bump, @cmccully. 

@embray, is this ok to merge? Looks like the only concern in this, #130 and #131 that hasn't been addressed is the potential issue with the release scripts, and I gather that isn't an insurmountable problem.

**Comment by pllim** [comment]
I like this as well but it looks like this PR needs rebasing.

**Comment by cmccully** [comment]
@pllim Can this be closed?

**Comment by pllim** [comment]
@cmccully, don't you want to rebase to get this merged?

**Comment by cmccully** [comment]
Oh. Yes I do. Sorry. I saw the merged thing above and misread it. I will rebase momentarily.

**Comment by cmccully** [comment]
@pllim I rebased this PR. Please let me know if you need anything else from me. Thanks.

File: setup.cfg [hunk 1]
```diff
@@ -25,6 +25,6 @@ license = BSD
 url = http://astropy.org/
 edit_on_github = False
 github_project = astropy/astropy
+# version should be PEP386 compatible (http://www.python.org/dev/peps/pep-0386)
+version = 0.0.dev
 
-[entry_points]
-astropy-package-template-example = packagename.example_mod:main
```
Comment (pllim) [inline setup.cfg]: This should not be removed. Must be an artifact from rebasing. Please add it back in.

**Comment by cmccully** [comment]
@pllim Sorry about that. I added back in the lines in question.

**Comment by pllim** [comment]
:+1: Looks good to me. Thanks! I hope package maintainer would merge this soon. I have been using your solution here for some of my work. :smile:

**Comment by astrofrog** [comment]
👍

**Comment by astrofrog** [comment]
@cmccully - can you rebase? (I merged the PR to remove Python 2.6 from Travis)

**Comment by cmccully** [comment]
Done @astrofrog. Thanks!

**Pull request merged**